include/envoy/router/router.h

```diff
@@ -843,6 +850,18 @@ class RouteEntry : public ResponseEntry {
    */
   virtual bool includeVirtualHostRateLimits() const PURE;
 
+  /**
+   *
+   * @return VhRateLimitOptions return the virtual rate limit option set for the route
+   */
+  virtual VhRateLimitOptions virtualHostRateLimitsOption() const PURE;
+
+  /**
+   *
+   * @return VhRateLimitOptionsCase return the type of virtual host rate limits option being used.
```

Comment:
Perhaps would make sense to note this is also deprecated and was added to support b/c with a deprecated config option.